src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/WindowsRazorProjectHostBase.cs

```diff
@@ -64,11 +64,6 @@ protected WindowsRazorProjectHostBase(
 
     protected sealed override Task InitializeCoreAsync(CancellationToken cancellationToken)
     {
-        if (_languageServerFeatureOptions.UseRazorCohostServer)
```

Comment:
Should we add an assert here as well?

Reply:
I guess it can't hurt. Realistically we could add an assert almost everywhere :D

Reply:
It's just in the context of this PR it looks like a place that was missed. Without this PR I probably wouldn't have second guessed at all